gfx.h

```diff
@@ -53,6 +53,7 @@ typedef uint32_t GfxCreateContextFlags;
 GfxContext gfxCreateContext(HWND window, GfxCreateContextFlags flags = 0, IDXGIAdapter *adapter = nullptr);
 GfxContext gfxCreateContext(uint32_t width, uint32_t height, GfxCreateContextFlags flags, IDXGIAdapter *adapter = nullptr);
 GfxResult gfxDestroyContext(GfxContext context);
+bool      gfxIsValid(GfxContext context);
```

Comment:
What about renaming this to `gfxHasDeviceCrashes()` or something along these lines?
`isValid` seems a bit vague imo, what do you think?

Reply:
Well it returns whether gfx has been initialized and whether the device is still working so its more than just a device crash check. So `gfxIsContextValid` or something else, ideas welcom

Reply:
Got it, then let's name it `gfxContextIsValid()` just to be more consistent with the rest of the naming; i.e., it's `gfxCreateBuffer()` and `gfxDestroyBuffer()` but any other buffer-related function is named `gfxBufferXxx()`.

Reply:
ok done